avocado/plugins/runner_nrunner.py

```diff
@@ -219,8 +220,12 @@ async def _update_status(self, job):
                                                         job.result,
                                                         test_state)
 
+                if not mapping[test_state['status']]:
+                    self.summary.add("FAIL")
```

Comment:
Is this summary attribute the correct place to store these statuses? This also relates to my question above.